app/views/user_journey/before_you_start.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= render 'shared/certificate_details', certificate: @certificate %>
+
+<h1 class="govuk-heading-l"> <%= t 'user_journey.before_you_start.title' %></h1>
+<p><%= t 'user_journey.before_you_start.rotate_certificate_read_steps' %></p>
+<p><%= t 'user_journey.before_you_start.before_you_upload' %><%= @certificate.component_type == COMPONENT_TYPE::MSA ? t('user_journey.before_you_start.msa_with_abbreviation') : t('user_journey.before_you_start.vsp_with_abbreviation') %>
```

Comment:
this could be potentially done in a shorter way...
`t 'user_journey.before_you_start.#{@certificate.component_type}.before_you_upload'` (obviously would need changing the locale file, but it would remove the need for the if statement here)